dev.yml

```diff
@@ -10,12 +10,8 @@ up:
   - docker
   - custom:
       name: Docker Compose
-      met?: docker compose ls | grep -ioE -q "lhm.*running\(3\)"
-      meet: docker compose up -d
-  - custom:
-      name: Waiting for DBs to be operational
       met?: ./scripts/helpers/wait-for-dbs.sh
-      meet: ":"
+      meet: docker-compose up -d
```

Comment:
The above two steps felt like they could be merged into one. I mean in the end after setting up docker-compose we want to see if the local test containers are up or not.

Reply:
Did you try to run this with the docker-compose being down? Or did you run it when it was already up?